CHANGELOG_PENDING.md

```diff
@@ -26,6 +26,8 @@ Friendly reminder, we have a [bug bounty program](https://hackerone.com/tendermi
 
 ### IMPROVEMENTS:
 
+- [log] [\#2765](https://github.com/tendermint/tendermint/issues/2765) Make "Update to validators" msg value pretty
```

Comment:
You can just do `\#2765` and we'll linkify it before release. We have a script for that :)